src/core/Table/TableRowMemoized.tsx

```diff
@@ -14,7 +14,7 @@ import { TableCell } from './TableCell';
  * Although state is not used it is needed for `React.memo` to check state that changes row state e.g. selection.
  * When adding new features check whether it changes state that affects row. If it does then add equality check to `React.memo`.
  */
-const TableRow = <T extends Record<string, unknown>>(props: {
+export const TableRow = <T extends Record<string, unknown>>(props: {
```

Comment:
why export now?

Reply:
Idk, it was in Mayank's PR so I did the same 😀 We also do that in other components.